.github/workflows/account-migration.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         uses: LedgerHQ/ledger-live/tools/actions/composites/setup-git-user@develop
 
       - name: Setup the toolchain
-        uses: LedgerHQ/ledger-live/tools/actions/composites/setup-toolchain@develop
+        uses: LedgerHQ/ledger-live/tools/actions/composites/setup-toolchain@fix/upgrade-pnpm
```

Comment:
do you keep this change for develop too ?

Reply:
I put this PR under review too early, sorry. I want to make the CI work with the updated workflows as we pin pnpm for some actions. Then I will drop the commit with this change before merging into develop